front/ui/ui-charts/src/manchette/hooks/useManchetteWithSpaceTimeChart.tsx

```diff
@@ -366,6 +366,7 @@ const useManchetteWithSpaceTimeChart = ({
     // Constant scale:
     if (isProportional) {
       const baseScale = baseScales[0];
```

Comment:
Can you use `at` ?
```suggestion
      const baseScale = baseScales.at(0);
```

Reply:
What's the motivation for this change?

Reply:
We discussed this out-of-band. `at()` returns a value which can be `undefined` while bracketed array access does not (because of `noUncheckedIndexedAccess`).

I've replaced this with a length check before the array access.